packages/rke2-cilium/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: https://helm.cilium.io/cilium-1.12.4.tgz
```

Comment:
the chart version needs to be bumped to 1.12.5 as per https://github.com/rancher/rke2/pull/3802/files#r1073763146.

When doing this, please reset the packageVersion to 00.